lib/store/watch.js

```diff
@@ -18,7 +18,8 @@ export default class WatchStore {
     this.editor = new TextEditor({
       softWrapped: true,
       grammar: this.kernel.grammar,
-      lineNumberGutterVisible: false
+      lineNumberGutterVisible: false,
+      placeholderText: `Enter ${kernel.displayName} code`
```

Comment:
To see the rendering issue i've referred to, start two different kernels and add some watches to each. If you tab back and forth you will see this text not update properly. 

I'll open a separate issue about this since the underlying problem is in master as well.